scripts/analyze_seasonal_coefficients.py

```
@@ -0,0 +1,340 @@
+        # 3. Весной больше азота для роста
+        # 4. Осенью больше фосфора для подготовки к зиме
+        
+        current = {
+            "spring": {"nitrogen": 1.15, "phosphorus": 1.10, "potassium": 1.12},
+            "summer": {"nitrogen": 1.08, "phosphorus": 1.05, "potassium": 1.20},
+            "autumn": {"nitrogen": 1.06, "phosphorus": 1.12, "potassium": 1.15},
+            "winter": {"nitrogen": 0.85, "phosphorus": 1.08, "potassium": 1.10}
+        }
+        
+        principles_met = 0
+        total_principles = 4
```

Ruff (0.12.2)

Local variable `total_principles` is assigned to but never used

Remove assignment to unused variable `total_principles`

(F841)